pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -500,6 +500,11 @@ func (cm *containerManagerImpl) GetNodeConfig() NodeConfig {
 	return cm.NodeConfig
 }
 
+// GetPodCgroupRoot returns the cgroup containing all pods
+func (cm *containerManagerImpl) GetPodCgroupRoot() string {
+	return cm.cgroupRoot
```

Comment:
this will return /kubepods w/ systemd cgroup driver which is wrong.

Reply:
if we are returning the literal cgroupfs value, you need to take into account the driver.

as a result, need to return

```
return cm.cgroupManager.Name(CgroupName(cm.cgroupRoot))
```

Reply:
done